lib/paratii.utils.js

```diff
@@ -0,0 +1,51 @@
+const crypto = require('crypto')
+const Web3 = require('web3')
+const dopts = require('default-options')
+
+export class ParatiiUtils {
+  constructor (config) {
```

Comment:
there is no need to copy the whole constructor from paratii. Probably, having:

    constructor (config) {
        this.config = config
   } 

should be good enough for purposes.  The web3 instance (note that there should be only one single web3 instance used throughot the whole app, which is created in `paratiiEth.js`) can then be recovered by `this.config.paratii.eth.web3` .